client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -408,9 +418,25 @@ float getShadow(sampler2D shadowsampler, vec2 smTexCoord, float realDistance)
 	return visibility / max(n, 1.0);
 }
 
+#endif
 #endif
 
+#if (defined(ENABLE_BUMPMAPS) && !defined(MATERIAL_LIQUID))
+//This is mostly a placeholder and probably should use proper textures eventually...
```

Comment:
Let's leave the bump mapping out of this PR.

Reply:
Fair

Reply:
I hope this is not annoying.
Bump mapping seems to be something we want to "right". (We also had it in the past and removed it.)

Reply:
I mean it's kind of a pain to undo, and I do think it's a cool effect, but I 100% see why this is a problem...